test/project_types/rails/gem_test.rb

```diff
@@ -14,6 +14,7 @@ def setup
     def teardown
       @context.setenv('GEM_HOME', nil)
       @context.setenv('GEM_PATH', nil)
+      super
```

Comment:
For missing `super` calls in `teardown`, I don't know of an automated way of finding these, I just manually checked them.

Reply:
This is a good question. If minitest creates a new test instance on every run there won't be any state left over on subsequent runs, so the only place I can think of would be to check the flag during garbage collection (if that's even possible).

Reply:
We could maybe use an `at_exit` hook.

But I'd like for this to be a stopgap measure, and use a linter instead.

I've added a entry to the minitest style guide:
https://github.com/rubocop-hq/minitest-style-guide/pull/35

So the next step would be to add it in rubocop-minitest:
https://github.com/rubocop-hq/rubocop-minitest/issues/111